snow/engine/snowman/bootstrap/storage.go

```diff
@@ -133,6 +134,12 @@ func execute(
 	tree *interval.Tree,
 	lastAcceptedHeight uint64,
 ) error {
+
+	if tree.Len() > minBlocksToCompact {
```

Comment:
```suggestion
	totalNumberToProcess := tree.Len()
	if totalNumberToProcess > minBlocksToCompact {
```
(And then we can delete the `totalNumberToProcess` below here)